server/fakedb/Event.csv

```diff
@@ -0,0 +1,12 @@
+﻿createdAt,updatedAt,hostId,title,description,price,maxParticipants
```

Comment:
Usually creationTIme, lastUpdatedTime is a better naming convention. updatedAt can be misleading when there are multiple updates.

Reply:
Thank you! I modified `schema.js`(GraphQL Schema) according to your comment.
But in database(`database.js`), These keywords('createdAt', 'updatedAt') are defined by Sequlize
(https://sequelize.org/v5/manual/models-definition.html#Timestamps)
If we create other fields like creationTime, lastUpdatedTime, we have to deal with these fields ourselves.
Isn't it inefficient to do that?

Reply:
Hmm I'm surprised that it forces you to use that name. This was just a suggestion, so up to you.

Reply:
Okay, Thank you!